index.html

```diff
@@ -1,10 +1,33 @@
-<h1>Promesse Seuglong Fongba</h1>
-<h2>About</h2>
-<p>I am from Cameroon. Iam happily married since 2017 with two beautiful kids. I am looking for a job.</p>
-<h2>expereience</h2>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Promesse Seuglong</title>
+</head>
+<body>
+    
+    <h1>Promesse Seuglong Fongba</h1>
```

Comment:
There is no reason for this h1 element to be outside the div wrapper; put it inside. This will allow you to remove the text-align: center CSS styling for the h1 tag.